src/pluginrc/saltplugins.rc

```diff
@@ -1,17 +1,16 @@
 #!/bin/bash
 
 function check_packages() {
-    RPMS=$1
-    for pkg in $RPMS; do
-	if ! rpm -q $pkg &>/dev/null; then
-	    echo "ERROR: Salt package \"$pkg\" is not installed"
-	    echo
-	    exit 111
+    for pkg in "$@"; do
+	if ! validate_rpm_pkg $pkg; then
+	    exit 111 
 	fi
     done
 }
 
 validate_rpm_pkg() {
```

Comment:
Nitpick: for consistency, can we use the same style for function declarations? I know you did not add this, but since you touch the file, do you mind adding a leading `function` statement?

Reply:
`function` statement included in the next commit and the function content returned to the original state.
I just misunderstood the initial purpose of the function. I supposed these two functions are related, but actually not.
`validate_rpm_pkg` is used only once in `saltgeneral` and it was not supposed to be used as a replacement for `rpm -q`. Just misunderstand the initial idea.

Reply:
Thanks for adding `function`!